elastic/beats
# [Filebeat] Add metrics for filestream

## What does this PR do?

Adds metrics to the filestream input.

## Why is it important?

Provides useful information for observability and diagnostic purposes.

## Checklist


## How to test this PR locally

- Ensure that metrics http endpoint is enabled.
- `curl -sL http://localhost:5066/inputs | jq '.[] | select(.input=="filestream")'`

## Related issues


- Closes elastic/security-team#6588

## Logs

```
{
  "bytes_processed_total": 75,
  "events_in_flight": 0,
  "events_processed_total": 15,
  "files_active": 1,
  "files_closed_total": 2,
  "files_created_total": 3,
  "files_deleted_total": 1,
  "files_ignored_total": 1,
  "files_read_total": 3,
  "files_renamed_total": 1,
  "files_truncated_total": 1,
  "files_updated_total": 1,
  "id": "test-file-1",
  "input": "filestream",
  "messages_read_total": 16,
  "processing_errors_total": 0,
  "processing_time": {
    "histogram": {
      "count": 15,
      "max": 503895,
      "mean": 56446.333333333336,
      "median": 13196,
      "min": 11427,
      "p75": 36137,
      "p95": 503895,
      "p99": 503895,
      "p999": 503895,
      "stddev": 122054.97778605984
    }
  }
}
```
